data/marinowijay.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "Marino Wijay",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**DevRel Dojo** | Platform Advocate @ Solo.io | DevOps | Kubernetes | Service Mesh | Cloud-Native Open Source",
+    "tags": ["DevRel" "Cloud-Native Open Source", "DevOps", "Kubernetes", "Service Mesh", "Networking"],

```

Comment:
comma missing here after `DevRel` in tags

Reply:
Thank you! Fixed